From dac1d64ee8ad61bf39a59365ffb7d4786b18ff92 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@gmail.com>
Date: Tue, 10 Oct 2017 10:36:10 +0800
Subject: [PATCH 895/895] cpufreq: ti-cpufreq: add missing of_node_put()

call of_node_put to release the refcount of np.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/cpufreq/ti-cpufreq.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/cpufreq/ti-cpufreq.c b/drivers/cpufreq/ti-cpufreq.c
index 5e6f7d2..fbbfed1 100644
--- a/drivers/cpufreq/ti-cpufreq.c
+++ b/drivers/cpufreq/ti-cpufreq.c
@@ -227,6 +227,7 @@ static int ti_cpufreq_probe(struct platform_device *pdev)
 
 	np = of_find_node_by_path("/");
 	match = of_match_node(ti_cpufreq_of_match, np);
+	of_node_put(np);
 	if (!match)
 		return -ENODEV;
 
-- 
1.7.5.4

